performance/utils.js

```diff
@@ -31,6 +31,13 @@ const clsValue = () => {
 	return page.evaluate(() => window.cls);
 };
 
+const newPageMultiple = async () => {
+	const newPage = await testMultiple.newPage();
```

Comment:
remove `targetEnv` and add `testMultiple` to global list variables
![image](https://user-images.githubusercontent.com/63335823/195343409-ba509c6d-1a00-4955-a451-a72c8b0f3226.png)